language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/document/BallerinaDocumentService.java

```diff
@@ -480,7 +480,8 @@ public CompletableFuture<ExecutorPositionsResponse> executorPositions(BallerinaP
                 if (module.isEmpty()) {
                     return response;
                 }
-                response.setExecutorPositions(ExecutorPositionsUtil.getExecutorPositions(module.get(),
+                response.setExecutorPositions(ExecutorPositionsUtil.getExecutorPositions(workspaceManagerProxy.get(),
```

Comment:
Should we pass the `module` now? Since we are passing both the `workspaceManager` and the `filePath`, `module` can be derived.

Reply:
+1 to remove